src/SFA.DAS.EAS.Portal.UnitTests/Portal/Application/Adapters/CohortAdapterTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using SFA.DAS.EAS.Portal.Application.Adapters;
+using SFA.DAS.EAS.Portal.UnitTests.Builders;
+
+namespace SFA.DAS.EAS.Portal.UnitTests.Portal.Application.Adapters
+{
+    [Parallelizable]
+    [TestFixture]
+    public class CohortAdapterTests
+    {
+        private readonly CohortAdapter _sut;
+
+        public CohortAdapterTests()
+        {
+            _sut = new CohortAdapter();
+        }
+
+        [Test]
+        public void WhenConvertCalled_ThenTheEventIsMappedToTheCommand()
+        {
+            // arrange
+            CohortApprovalRequestedByProvider @event = new CohortApprovalRequestedByProviderBuilder();
+            var accountId = @event.AccountId;
```

Comment:
Fixed 👍.

As an aside, in situations like this, I tend to clone the object, and use the cloned object for the expected properties. It's neither here nor there for an object with 3 properties, but can be useful for more complex objects (and a general pattern).

Reply:
Nice, clone added 👍.